<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hand_picked_fibers_widget</class>
 <widget class="QWidget" name="hand_picked_fibers_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>423</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit Overlay</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDoubleSpinBox:disabled {background-color: LightGray;}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="fiber_step_label">
     <property name="text">
      <string>Fiber Step:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="ScientificDoubleSpinBox" name="fiber_step">
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="suffix">
      <string>°</string>
     </property>
     <property name="decimals">
      <number>8</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.500000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="current_group">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Current Fiber</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_15">
      <item row="5" column="0" colspan="3">
       <layout class="QHBoxLayout" name="current_orientation_layout">
        <item>
         <widget class="ScientificDoubleSpinBox" name="current_orientation_0">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="decimals">
           <number>6</number>
          </property>
          <property name="minimum">
           <double>-10.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ScientificDoubleSpinBox" name="current_orientation_1">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="decimals">
           <number>6</number>
          </property>
          <property name="minimum">
           <double>-10.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ScientificDoubleSpinBox" name="current_orientation_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="keyboardTracking">
           <bool>false</bool>
          </property>
          <property name="decimals">
           <number>6</number>
          </property>
          <property name="minimum">
           <double>-10.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QPushButton" name="add_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Add Fiber</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="current_label">
        <property name="text">
         <string>Right-click the canvas to generate fibers for a position</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QSlider" name="current_slider">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>719</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="ScientificDoubleSpinBox" name="current_angle">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string>°</string>
        </property>
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="picked_group">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Picked Fibers</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_16">
      <item row="0" column="0">
       <widget class="QTableWidget" name="picked_table">
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="delete_selected">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Delete Selected</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScientificDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>scientificspinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>fiber_step</tabstop>
  <tabstop>current_slider</tabstop>
  <tabstop>current_angle</tabstop>
  <tabstop>current_orientation_0</tabstop>
  <tabstop>current_orientation_1</tabstop>
  <tabstop>current_orientation_2</tabstop>
  <tabstop>add_button</tabstop>
  <tabstop>picked_table</tabstop>
  <tabstop>delete_selected</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
